utils_cv/similarity/references/evaluate.py

```diff
@@ -0,0 +1,156 @@
+# Most of the code in this file is copied and slightly modified from:
+# https://github.com/layumi/Person_reID_baseline_pytorch/blob/master/evaluate.py
+
+import numpy as np
+import time
+import torch
+from .re_ranking import re_ranking
+
+
+def evaluate(ds, features, use_rerank=False):  # labels, features
```

Comment:
my only suggestion, as discussed on the call, is to pull this function out of `references` and add a unit test for this.

Reply:
and just make sure it conforms to our standards (ie, using type-hints)